sim/common/sod/crafted/phase_2.go

```diff
@@ -113,9 +113,11 @@ func init() {
 	// Glowing Gneuro-Linked Cowl
 	core.NewItemEffect(215166, func(agent core.Agent) {
 		character := agent.GetCharacter()
+		actionId := core.ActionID{SpellID: 437349}
```

Comment:
Implemented the damage portions of these items as they contribute to getting a higher Berserking value